Makefile

```diff
@@ -143,6 +143,7 @@ dist-clean: clean
 	$(MAKE) -C $(KOR_BASE) dist-clean
 	$(MAKE) -C doc clean
 
+ZIP_EXCLUDE=-x "*.swp" -x "*.swo" -x "*.orig"
```

Comment:
Added, I don't even know you can do persistent undo file in vim, LOL. How do you clean up those files? Looks like they lives forever?